src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/EnclaveSessionCache.NetCoreApp.cs

```diff
@@ -69,7 +69,7 @@ internal SqlEnclaveSession CreateSession(string attestationUrl, string serverNam
         // Generates the cache key for the enclave session cache
         private string GenerateCacheKey(string serverName, string attestationUrl, string database)
         {
-            return (serverName + database + attestationUrl).ToLowerInvariant();
+            return (serverName + '+' + database + attestationUrl).ToLowerInvariant();
```

Comment:
Can a database name be case sensitive?

Reply:
No, it is not. You can test it from SSMS.